recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
> Oops -- but now I'm confused. I thought they are evaluated like Python expressions 

I believe they are.

> @patricksnape what is the proper flag to get a boolean `and`?

If your goal is to `skip` `Python 3` only on `OS X` that flag is correct. If you mean to skip Python 3 on all platforms while also skipping `OS X ` entirely you should use `[py3k or osx ]`. Does this helps you?